src/js/player.js

```diff
@@ -2473,6 +2477,32 @@ class Player extends Component {
     return this.techGet_('readyState');
   }
 
+  /**
+   * Get a video track list
+   * @link https://html.spec.whatwg.org/multipage/embedded-content.html#videotracklist
+   *
+   * @return {VideoTrackList} thes current video track list
+   * @method videoTracks
+   */
+  videoTracks() {
+    // cannot use techGet_ directly because it checks to see whether the tech is ready.
+    // Flash is unlikely to be ready in time but videoTracks should still work.
+    return this.tech_ && this.tech_['videoTracks']();
+  }
+
+  /**
+   * Get an audio track list
+   * @link https://html.spec.whatwg.org/multipage/embedded-content.html#audiotracklist
+   *
+   * @return {AudioTrackList} thes current audio track list
+   * @method audioTracks
+   */
+  audioTracks() {
+    // cannot use techGet_ directly because it checks to see whether the tech is ready.
+    // Flash is unlikely to be ready in time but audioTracks should still work.
+    return this.tech_ && this.tech_['audioTracks']();
```

Comment:
For background, we didn't do that for TextTracks because you probably shouldn't be calling these methods on a player that isn't ready (and thus with a tech that isn't ready) plus since the lists are stored on the tech, if we return an empty track list, we'll need to make sure that it is cached and that the tech that is loaded uses that instead of making a new one because otherwise the events like `addtrack` won't work for that one-off.